core/coreapi/name.go

```diff
@@ -170,7 +194,7 @@ func keylookup(self ci.PrivKey, kstore keystore.Keystore, k string) (ci.PrivKey,
 			return nil, err
```

Comment:
Why is it not doing IDFromPrivateKey(privKey) directly?

Reply:
Just a couple of lines above, could not add comment directly there.

Reply:
No idea. It may not have existed? I'll change that.